src/VisualStudio/CSharp/Impl/Options/IntelliSenseOptionPageControl.xaml.cs

```diff
@@ -57,31 +52,6 @@ private void Show_completion_list_after_a_character_is_typed_Unchecked(object se
         {
             Show_completion_list_after_a_character_is_deleted.IsEnabled = Show_completion_list_after_a_character_is_typed.IsChecked == true;
             Show_completion_list_after_a_character_is_deleted.IsChecked = false;
-            Show_completion_list_after_a_character_is_deleted_Unchecked(sender, e);
-        }
-
-        private void Show_completion_list_after_a_character_is_deleted_Checked(object sender, RoutedEventArgs e)
```

Comment:
`Show_completion_list_after_a_character_is_deleted_Checked` and `Show_completion_list_after_a_character_is_deleted_Unchecked` could be removed because on line 22
`BindToOption(Show_completion_list_after_a_character_is_deleted, CompletionOptionsStorage.TriggerOnDeletion, LanguageNames.CSharp, onNullValue: () => false);`